Vagrantfile

```diff
@@ -79,6 +79,16 @@ local_config = {
   "source_root" => (ENV['SOURCE_ROOT'] || '/vagrant'),
 }
 
+class Hash
+  def slice(*keep_keys)
+    h = {}
+    keep_keys.each { |key| h[key] = fetch(key) if has_key?(key) }
+    h
+  end unless Hash.method_defined?(:slice)
+  def except(*less_keys)
+    slice(*keys - less_keys)
+  end unless Hash.method_defined?(:except)
+end
```

Comment:
this spooky monkey patching definately deserves a comment or link to the bug or stack overflow or whatever